Makefile

```
@@ -0,0 +1,32 @@
+.DEFAULT_GOAL := help
+
+KUBECONFIG ?= $(shell echo "${KUBECONFIG}")
+CHART ?= free-stuff-matrixbot
+VERSION ?= 0.0.1
+RELEASE ?= free-stuff-matrixbot
+VALUES ?= values.yml
+SECRETS_ENV_FILE ?= .env
+DEBUG ?= --debug
+DRYRUN ?=
+
+## Show available make targets
+.PHONY: help
+help:
+	@echo "$(notdir $(realpath .)) make targets:"
+	@$(MAKE) -pRrq -f $(lastword $(MAKEFILE_LIST)) : 2>/dev/null | awk -v RS= -F: '/^# File/,/^# Finished Make data base/ {if ($$1 !~ "^[#.]") {print $$1}}' | sort | egrep -v -e '^[^[:alnum:]]' -e '^$@$$'
+
+.PHONE: apply-secret
```

If this is intended and I just didn't know about `.PHONE`, I'm happy to learn new things
```suggestion
.PHONY: apply-secret
```